app/workers/rapid_ready_for_decision/form526_base_job.rb

```diff
@@ -32,6 +32,7 @@ def perform(form526_submission_id)
         message = "Sidekiq job id: #{jid}. The error was: #{e.message}.<br/>" \
                   "The backtrace was:\n #{e.backtrace.join(",<br/>\n ")}"
         form526_submission.send_rrd_alert_email('Rapid Ready for Decision (RRD) Job Errored', message)
+        form526_submission.save_metadata(error: message)
```

Comment:
Seems okay either way, but just checking if this should be `e.message` so that it doesn't have the backtrace?

Reply:
Good point! Changed.